aspnetcore/host-and-deploy/docker/building-net-docker-images.md

```diff
@@ -120,9 +120,10 @@ The sample Dockerfile uses the [Docker multi-stage build feature](https://docs.d
 
 * Press Ctrl+C at the command prompt to stop the app.
 
-## Run in a Linux container
+## Run in a Linux container or Windows container
 
-* In the Docker client, [switch to Linux containers](https://docs.docker.com/desktop/windows/#switch-between-windows-and-linux-containers).
+* To run in a Linux container, in the Docker client, [switch to Linux containers](https://docs.docker.com/desktop/windows/#switch-between-windows-and-linux-containers).
```

Comment:
There are two UI's involved here, the Docker app, and then the Docker client icon in the System Tray.   Making that clear for new Docker users so they don't spend time looking around the app UI for this.
```suggestion
* To run in a Linux container, right-click the System Tray's Docker client icon and select, [switch to Linux containers](https://docs.docker.com/desktop/windows/#switch-between-windows-and-linux-containers).
```